src/dmd/toobj.d

```diff
@@ -968,7 +968,7 @@ void toObjFile(Dsymbol ds, bool multiobj)
             Symbol *tlvInit = symbol_name(tlvInitName, SCstatic, t);
             tlvInit.Sdt = null;
             tlvInit.Salignment = type_alignsize(s.Stype);
-            if (vd.linkage == LINK.cpp)
+            if (vd._linkage == LINK.cpp)
```

Comment:
This is ok, because I doubt target linkage would be cpp, but shouldn't all codegen parts use resolvedLinkage?

Reply:
Yeah, I was a bit lazy for multiple usage sites, assuming the system-linkage is never C++, D, nor Objective-C.